README.md

```diff
@@ -1290,7 +1290,20 @@ is expected to correspond to a SQLAlchemy table object of the same name, case in
 schema type this could look like:
 
 ```python
-from sqlalchemy import MetaData, Table, Column, String
+from graphql import parse
```

Comment:
I don't understand what these documentation changes have to do with the bug this PR originally set out to fix.